package.json

```diff
@@ -14,6 +14,7 @@
     "lib-esm"
   ],
   "scripts": {
+    "build": "npm run tsc && cheminfo-build",
```

Comment:
@josoriom did you try to run it locally ? I think you need to specify the entry point and you may also specify a top level variable.

Something like:

https://github.com/mljs/spectra-processing/blob/master/package.json#L14